src/ILLink.RoslynAnalyzer/Resources.resx

```diff
@@ -117,10 +117,10 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="RequiresUnreferencedCodeAnalyzerTitle" xml:space="preserve">
-    <value>RequiresUnreferencedCodeAnalyzer</value>
+  <data name="RequiresUnreferencedCodeTitle" xml:space="preserve">
+    <value>Avoid calling members marked with `RequiresUnreferencedCodeAttribute` when trimming application code</value>
   </data>
-  <data name="RequiresUnreferencedCodeAnalyzerMessage" xml:space="preserve">
+  <data name="RequiresUnreferencedCodeMessage" xml:space="preserve">
     <value>Calling '{0}' which has `RequiresUnreferencedCodeAttribute` can break functionality when trimming application code. {1}. {2}</value>
```

Comment:
Since you're editing things here, I think we should update this message to match the one we produce from within the linker: https://github.com/mono/linker/pull/1778#discussion_r564429652